strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,45 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxFileSizeString) throws SizeFormatValidationException
+    {
+
+        if (this.multipartConfigElement instanceof UpdatableMultipartConfigElement)
+        {
+            Long parsedSize = parseUploadSize(maxFileSizeString);
+            ((UpdatableMultipartConfigElement) multipartConfigElement).setMaxFileSize(parsedSize);
+            
+            return true;
+        }
+
+        return false;
+    }
+
+    private Long parseUploadSize(String maxFileSizeString) throws SizeFormatValidationException
+    {
+
+        maxFileSizeString = maxFileSizeString.toUpperCase();
+        String sizeUnit = maxFileSizeString.replaceAll("[^A-Z]", "");
+        Long size = Long.parseLong(maxFileSizeString.replaceAll("[^0-9]", ""));
+
+        int pow;
+        switch (sizeUnit){
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
        switch (sizeUnit)
        {
```